### PR TITLE
Style guide updates

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

Working on some high-level updates to the style guide

### Reviewer guidance

Please read and give feedback. Might need to preemptively merge in and make updates in another PR. 

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+    <p>
+      At Learning Equality, one of our
+      <a href="https://learningequality.org/about/values/">core values</a> is to
+      "design for everyone". Kolibri should be accessible to the widest possible
+      audience, in the broadest sense of the word 'accessible'. It should run on
+      wide variety of devices, render correctly on a wide variety of screen sizes
+      and assistive technologies, be understandable to people from a wide variety
+      of backgrounds, and be usable by people with a wide variety of abilities and
+      disabilities.
```
Comment (radinamatic) [inline kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue]: Maybe change `a wide variety of abilities and disabilities.` for `diverse abilities`.

File: kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      Material Design. These are exposed in our app by the
+      <code>responsiveWindow</code> Vue component mixin. We should never use CSS
+      media queries.
+    </p>
+
+    <h2>Keyboard navigation</h2>
+
+    <p>
+      Every interaction that can be performed with a mouse or touchscreen should be
+      accessible with only the keyboard. As much as possible, we should follow best
```
Comment (radinamatic) [inline kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue]: Maybe change `with only` for `also by using`.

File: kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+      readers. Additionally, the app should leverage screen-reader-specific
+      functionality – such as labels, semantically correct HTML tags, and
+      visually-hidden text – that associates descriptive text with functionality.
+    </p>
+
+    <p>
+      Like with keyboard navigation, using the built-in components is a good way to
+      ensure compatibility with screen readers. However, it's also critical to test
+      all new functionality. See this
+      <a href="https://github.com/learningequality/ka-lite/wiki/Accessibility-Resources-(Tools)">
```
Comment (radinamatic) [inline kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue]: There is an updated list on Kolibri's own Wiki 🙂 
https://github.com/learningequality/kolibri/wiki/Accessibility-Resources-(Tools)

File: kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+      comprehensive list of A11y testing tools</a>.
+    </p>
+
+    <p>
+      Some specific things to watch for:
+    </p>
+
+    <ul>
+      <li>Use ARIA attributes when the HTML semantics are not sufficient</li>
```
Comment (radinamatic) [inline kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue]: Change to : `<li>Use ARIA attributes <strong>ONLY</strong> when the HTML semantics are not sufficient</li>`

File: kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+      <li>
+        Nest header tags properly: <code>H1</code>, <code>H2</code>,
+        <code>H3</code>, etc...
+      </li>
+      <li><code>alt</code> text on headers</li>
```
Comment (radinamatic) [inline kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue]: `alt` text on headers? Need more context here... 😕

Comment (indirectlylit) [reply kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue]: whoops that was supposed to say images

**Comment by indirectlylit** [comment]
@radinamatic @jtamiace thanks for the review! changes addressed

@jtamiace - you were right, the link was broken

**Pull request merged**